source/module_hamilt_lcao/module_gint/gint_tools.h

```diff
@@ -284,8 +284,7 @@ namespace Gint_Tools
         const int*const block_size,
         bool** cal_flag,
         double** psi,
-		double** psi_DMR,
-        double* DMR,
```

Comment:
I am not sure this is a good idea, in fact I think passing DMR into every function may be a better choice than defining DMR in some class

Reply:
The "non-used" does not mean that `DMR` is a member of the class `Gint_Tools`, but means that `DMR` doesn not appear in the function at all.